src/event-helper.js

```diff
@@ -40,6 +40,32 @@ export function listen(element, eventType, listener, opt_capture) {
 
 
 /**
+ * Globally Listens for the specified event for the specified tag name.
+ * @param {!Window} win
+ * @param {string} tagName
+ * @param {string} eventType
+ * @param {function(Event)} listener
+ * @param {boolean=} opt_capture
+ * @return {!UnlistenDef}
+ */
+export function listenForEventOnTagName(
+    win, tagName, eventType, listener, opt_capture) {
+  const capture = opt_capture || false;
+  win.addEventListener(eventType, event => {
+    if (event.target.tagName.toLowerCase() == tagName.toLowerCase()) {
+      listener(event);
+    }
+  }, capture);
+  return () => {
+    if (win) {
+      win.removeEventListener(eventType, listener, capture);
```

Comment:
Oh. You're right. I'd have to proxy this properly.

I actually prefer to use per-element click listener as it'll be more readable and easier to understand.